CIPs/cip-0051.md

```diff
@@ -0,0 +1,157 @@
+        { name: 'chainId', type: 'uint256'},  
+        { name: 'verifyingContract', type: 'address'}, 
+    ],
+    IdentifierOwnershipAttestation: [
+        { name: 'identifier', type: 'bytes32' },
+        { name: 'issuer', type: 'address'},
+        { name: 'account', type: 'address' },
+        { name: 'signer', type: 'address' },
+        { name: 'issuedOn', type: 'uint256' },
+    ],
+},
+primaryType: 'IdentifierOwnershipAttestation',
+domain: {
+  name: 'FederatedAttestations',
+  version: '1.0',
+  {chainId},
+  verifyingContract: {contractAddress}
+},
+message: {
+    identifier: string,
+    issuer: string,
+    account: string,
+    signer: string,
+    issuedOn: number,
+}
+```
+
+## Backwards Compatibility
+
+### Interoperability SDK
+
+To address the fragmentation of credentials stored in `Attestations.sol` vs. `FederatedAttestations.sol`, an SDK will be created to resolve access to attestations from both ASv1 and ASv2. Further details [here](https://www.notion.so/clabsco/Federated-Attestation-Protocol-ASv2-05dc4843139842768ad9fe192cb46c00#55e61742befe49e0a55fb38e6f223172).
+
+### Escrow
+
+The Escrow contract has been upgraded to be compatible with both ASv1 and ASv2. New escrowed payments can specify a list of trusted issuers whose attestations will be accepted to redeem payments. The trusted issuer list can include phone-number attestation issuers from ASv2 in addition to the address of the `AttestationProxy` stored in `Registry.sol` to represent the attestations from ASv1 contained in `Attestations.sol`. This will provide backwards compatibility and increased flexibility for dApp developers.
+
+## Further Work
+
+We aim to later introduce metadata attestations in addition to ownership attestations. This would enable further information about an identifier to be claimed. Examples include phone number type, Twitter verification status, compliance with ODIS, etc. Furthermore, a more extensive system was originally imagined, where issuers issue [verifiable credentials](https://www.w3.org/TR/vc-data-model/) and identifiers follow the [DID standard](https://www.w3.org/2019/08/did-20190828/). These credentials are custodied by the user, and optionally can be shared on-chain to be discoverable by other parties, or can be shared off-chain directly between stakeholders.
+
+To allow for users to easily decide which issuers to trust, there are plans for designing smart contracts and corresponding tooling to create [crowdsourced trusted issuer registries](https://www.notion.so/clabsco/Decentralisation-and-Governance-d673afd33dbe4f8d8b70daa2ef6d4763).
+
+We hope that the information in these attestations can later be used to create lightweight reputation and sybil resistance signals for various use cases.
+
+## Further Reading
+
+- [Federated Attestations Protocol](https://clabsco.notion.site/ASv2-On-chain-Registry-Protocol-47e7850656444c37973856e0634ee2f9) is the main high-level design doc with our intial overarching vision
```

Comment:
@0xarthurxyz hey one note on this, this actually links to the protocol design doc; we should probably update the link to point to https://clabsco.notion.site/Federated-Attestation-Protocol-ASv2-05dc4843139842768ad9fe192cb46c00
and then add another entry to point to the protocol design doc (linked)

Reply:
Thanks for catching that @eelanagaraj! 🕵️‍♀️